<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(255, 255, 255);</string>
  </property>
  <widget class="QFrame" name="verticalFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(50, 50, 50);</string>
   </property>
   <widget class="QFrame" name="horizontalFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>701</width>
      <height>211</height>
     </rect>
    </property>
    <widget class="QPushButton" name="deny_exist">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>300</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid rgb(0, 114, 177);
	border-radius: 5px;	
	background-color: rgb(0, 114, 177);
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(25, 158, 193);
	border: 2px solid rgb(25, 158, 193);
	color: rgb(255, 255, 255);
}
QPushButton:pressed {	
	background-color: rgb(25, 158, 193);
	border: 3px solid rgb(20, 126, 154);
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/cancel.png</normaloff>icons/cancel.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="accept_exist">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>300</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 2px solid rgb(0, 114, 177);
	border-radius: 5px;	
	background-color: rgb(0, 114, 177);
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(25, 158, 193);
	border: 2px solid rgb(25, 158, 193);
	color: rgb(255, 255, 255);
}
QPushButton:pressed {	
	background-color: rgb(25, 158, 193);
	border: 3px solid rgb(20, 126, 154);
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/icons8-checkmark-100.png</normaloff>icons/icons8-checkmark-100.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="text_dialog">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>561</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245,245,245);</string>
    </property>
    <property name="text">
     <string>Are you sure to quit?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="icon">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>300</width>
      <height>140</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
background-image: url(icons/question.png);
background-position: center;
background-repeat: no-repeat;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
